test/e2e/upgrade_clusterctl_test.go

```diff
@@ -44,7 +44,7 @@ var _ = Describe("When testing cluster upgrade from releases (v1.9=>current) [cl
 		os.Setenv("IMAGE_RAW_CHECKSUM", imageChecksum)
 		os.Setenv("IMAGE_RAW_URL", imageURL)
 		// We need to override clusterctl apply log folder to avoid getting our credentials exposed.
-		clusterctlLogFolder = filepath.Join(os.TempDir(), "target_cluster_logs", bootstrapClusterProxy.GetName())
+		clusterctlLogFolder = filepath.Join(artifactFolder, bootstrapClusterProxy.GetName())
```

Comment:
I think this is OK, but the comment above should be removed. We are not giving any special credentials to clusterctl that could be exposed in the logs.

Reply:
done